### ISSUE TITLE
[Bug] Program would close quietly after being minimized

### ISSUE THREAD

**Issue by ychen8777** [JasonStein/Notepads#660]
**Describe the bug**
Notepads would close quietly after a few seconds its window is minimized. And the process would disappear from Task Manager.

**To Reproduce**
Click minimize on the program window

**Expected behavior**
The window is minimized while stay in the program bar.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 10 Home, 18362.1016
 - Version: 1903

**Additional context**

**Comment by sktnetwork** [comment]
Yes, confirmed. Happens to me as well. Very annoying, I lost my work. Pretty bad.

Notepads Version v1.4.2.0

Edition	Windows 10 Pro
Version	20H2
Installed on	‎19-‎08-‎2020
OS build	19042.804
Experience	Windows Feature Experience Pack 120.2212.551.0

**Comment by Gilibenzio** [comment]
I also confirm the bug.
I lost my work too, luckily they were not important documents. I've noticed that if you enable the option of backup and restore the session, the program closes anyway, but when it reopens the session is restored as it should be.
I attach a short video of the bug.
I look forward to a resolution of the problem.


OS: Microsoft Windows 10 Pro
OS version: 2H20, 10.0.19042 N/A build 19042.804
Windows Feature Experience Pack 120.2212.551.0

https://user-images.githubusercontent.com/73967962/110708867-9dd9c080-81fb-11eb-9e28-92123004f01f.mp4

**Comment by JasonStein** [comment]
It's more like a Windows issue to me. I will keep it in mind to see if I can figure out the reason here.

**Comment by YakovL** [comment]
Not only after being minimized, it also gets closed after switching Windows virtual desktops back and forth. Like: edit stuff in notepads, win+ctrl+right to go to another desktop, do stuff there, win+ctrl+left to go back to the v.d. with notepads, but.. it is closed, the results of work are lost.

For me, this happens on Windows 10 Pro, 21H2 (19044.1706) with Notepads 1.4.8.0.

**Comment by YakovL** [comment]
PS this may be helpful for fixing: [1](https://social.msdn.microsoft.com/Forums/windowsapps/en-US/5c297b49-989c-4a3d-abe6-adb963d6f507/uwpapp-crashes-when-minimised-and-maximised-the-app?forum=wpdevelop), [2](https://github.com/microsoft/microsoft-ui-xaml/issues/3802)

**Comment by user9713** [comment]
This bug never got fixed, huh? It's been driving me crazy for a while now and questioning my sanity as to whether I closed the file or not.

Considering this is a known issue since 2022, I think this is the end for Notepads, at least for me.
